### ISSUE TITLE
hit detection

### ISSUE THREAD

**Issue by davidm-public** [nodebox/g.js#6]
hi, nice library, wondering if it's possible to do hit detection on a shape,
ie: to know which shape the mouse is over?

thanks

**Comment by fdb** [comment]
Yes, a `Path` object has a `.contains` method that takes an x/y argument. This also works on a `Group`.

**Comment by fdb** [comment]
Here's the source:

https://github.com/nodebox/g.js/blob/88c9deaba4d57855915264dec4bcf8054b230f17/src/libraries/vg/objects/path.js#L396
